AppCenter/AppCenter/Internals/Context/AuthToken/MSAuthTokenInfo.m

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#import "MSAuthTokenInfo.h"
+
+static NSString *const kMSAuthTokenKey = @"authTokenKey";
+static NSString *const kMSStartTimeKey = @"startTimeKey";
+static NSString *const kMSEndTimeKey = @"endTimeKey";
+
+@implementation MSAuthTokenInfo
+
+- (instancetype)initWithAuthToken:(nullable NSString *)authToken
```

Comment:
So it all can be `nil`? 🤔

Reply:
Yes

Reply:
This covers anonymous when it is `nil`. We should know from the history if a user sign-out.